### PR TITLE
Update runtime version from 42 to 44

### PR DESCRIPTION
**Pull request by eshimoniak**
Version 42 is end-of-life as of March 21, 2023. As far as I can tell this change doesn't introduce any problems, but if anyone can double check I would appreciate it.

### CODE REVIEW

File: io.github.jliljebl.Flowblade.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 app-id: io.github.jliljebl.Flowblade
 default-branch: stable
 runtime: org.gnome.Platform
-runtime-version: '42'
+runtime-version: '44'
```
Comment (hfiguiere) [inline io.github.jliljebl.Flowblade.yaml:4]: you need to change the plugin version to 22.08

Comment (hfiguiere) [reply io.github.jliljebl.Flowblade.yaml:4]: and remove Jack as it is in the SD

Comment (hfiguiere) [reply io.github.jliljebl.Flowblade.yaml:4]: and the python path changed to 3.10

File: io.github.jliljebl.Flowblade.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 app-id: io.github.jliljebl.Flowblade
 default-branch: stable
```
Comment (hfiguiere) [inline io.github.jliljebl.Flowblade.yaml:2]:
```suggestion
```

also let's remove this too.

File: io.github.jliljebl.Flowblade.yaml (hunk 1 above)
Comment (eshimoniak) [reply io.github.jliljebl.Flowblade.yaml:3]: Thanks for the input, I'll get on that now.

**Comment by hfiguiere** [comment]
the setup script seems to break with python 3.10.

**Pull request closed without merging**